include/gsl/multi_span

```diff
@@ -2019,96 +2058,90 @@ private:
 
     pointer data_;
     const Span* m_validator;
-    void validateThis() const
-    {
+
+    GSL_SUPPRESS(bounds.1) // NO-FORMAT: attribute
+    void validateThis() const {
         // iterator is out of range of the array
         Expects(data_ >= m_validator->data_ && data_ < m_validator->data_ + m_validator->size());
     }
+
+    GSL_SUPPRESS(bounds.1) // NO-FORMAT: attribute
     contiguous_span_iterator(const Span* container, bool isbegin)
         : data_(isbegin ? container->data_ : container->data_ + container->size())
         , m_validator(container)
-    {
-    }
+    {}
 
 public:
-    reference operator*() const GSL_NOEXCEPT
+    reference operator*() const
```

Comment:
Why is noexcept removed here? I know that `validateThis` is not noexcept, but noexcept is not removed for `operator ->` which looks quite similar.

Reply:
mistake in `operator->` ... will fix